package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@expo/react-native-action-sheet": "^1.1.0",
+    "@react-native-community/netinfo": "3.2.1",
```

Comment:
Please use a leading `^` -- that way a plain `yarn upgrade` in the future will get the latest 3.x.

(Notice that almost all our dependencies use that style.)

Reply:
Yup!

But I thought, there will not be any new 3.x.x release, as they already moved to 4.x.x

Anyway I agree it to have `^`, doing it 👍